docs/api/covidcast_changelog.md

```diff
@@ -44,6 +44,19 @@ Duplicate survey weights had corrupted historical figures for the following sign
 
 We now include figures on Puerto Rico for all `jhu-csse` signals at the state level.
 
+#### 1 September 2020
+
+NY Boroughs county FIPS (36005, 36047, 36061, 36081, 36085) are now split in proportion to the population of each county, instead of being reported in aggregate in FIPS 36061. 
+
+#### 30 September 2020
+
+The following changes were made to all `jhu-csse` signals related to geocoding
+- NY Boroughs county FIPS (36005, 36047, 36061, 36081, 36085) are now [differentiated by JHU](https://github.com/CSSEGISandData/COVID-19/issues/3084),
+- “Unassigned” and “Out of State” counts are assigned to a megaFIPS at the county level (XX000 where XX is the state FIPS code) and will now be incorporated into the state totals
+- The split of the Kansas City, Missouri into its four subcounties is now done with a more precise population proportion,
+- The split of the Dukes and Nantucket, Massachusetts FIPS codes into the two subcounties is now done with a more precise population proportion,
```

Comment:
In `jhu-csse.md`, the edit says this split is done evenly, not according to a population proportion -- which is it?